src/randoop/operation/MethodCall.java

```diff
@@ -94,40 +95,32 @@ public MethodCall(Method method) {
   }
 
   /**
-   * Returns the statement corresponding to the given constructor.
+   * getMethodCall a static method that creates a {@link MethodCall} corresponding 
```

Comment:
In the first word here a typo?
I don't think you need to state it is a static method -- that will be obvious to anyone who reads this documentation.  I would just start, "Creates a ...".
This documentation is much better than the misleading previous documentation.  However, the name is still confusing.  This is a creator (constructor) method, so I propose that its name should start with "create" or "make" or "new" rather than "get" which implies that it returns something that already exists.  Please see if any other methods should also be ranamed.
Also, this renaming and a few others appear on a disproportionate number of lines (at least in the part of the diff I have read so far).  If true, would it make sense to create a pull request for those renamings in order to simplify the other changes?